R/reportEmi.R

```diff
@@ -517,17 +517,25 @@ reportEmi <- function(gdx, output=NULL, regionSubsetList=NULL,t=c(seq(2005,2060,
   # following e.g. "Emi|CO2|Energy|Demand|Transport|Passenger|+|Liquids" etc.
   
   #### Total energy CO2 emissions
-  out <- mbind(out,
-               setNames(
-               # vm_emiTeMkt is variable in REMIND closest to energy co2 emissions
-               (dimSums(mselect(vm_emiTeMkt, all_enty="co2"), dim=3) 
-               # deduce vm_emiTeMkt by CCU CO2 from vm_emiCdr 
-               # (which is only deduced from vm_emiAll, the total CO2 emissions, in REMIND)
-               - (1-p_share_CCS)*mselect(-vm_emiCdr, all_enty="co2"))*GtC_2_MtCO2,
+  if(getSets(vm_emiTeMkt)[[3]] == "emiTe"){
+    out <- mbind(out,
+              setNames(
```

Comment:
The indentation is a little off here. I'd suggest in RStudio `ctrl+a` then `ctrl+i` to auto-indent everything, then `lucode2::autoFormat('R/reportEmi.R')` to remove unnecessary whitespace introduced by `ctrl+i`. This kind of auto-formatting can make it hard to see the actual changes, so I'd suggest you do this after everything else is done, so right before merging, in a separate commit.